enterprise/install-scripts/amazon/openshift-amz.sh

```diff
@@ -399,7 +401,7 @@ configure_rhn_channels()
     fi
     need_node_repo && repos+=('rhel-x86_64-server-6-ose-2.2-node' 'jb-ews-2-x86_64-server-6-rpm')
     need_client_tools_repo && repos+=('rhel-x86_64-server-6-ose-2.2-rhc')
-    need_jbosseap_cartridge_repo && repos+=('rhel-x86_64-server-6-ose-2.2-jbosseap' 'jbappplatform-6-x86_64-server-6-rpm')
+    need_jbosseap_cartridge_repo && repos+=("rhel-x86_64-server-${jbosseap_version:-6}-ose-2.2-jbosseap" "jbappplatform-${jbosseap_version:-6}-x86_64-server-6-rpm")
```

Comment:
Would ose jbosseap cartridge channel also have multiple version just like jbeap channel? As far as I know, seem like there is only one version - 6, am I right?